src/main/java/com/strandls/document/pojo/DocumentCoverageData.java

```diff
@@ -11,7 +11,9 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class DocumentCoverageData {
-
+	private Long id;
```

Comment:
why adding this field here , thiers already a class with all the field as DocumentCoverage??